### PR TITLE
Allow omitting more config values

### PR DESCRIPTION
**Pull request by lava**
###  :notebook_with_decorative_cover: Description

This PR allows users to omit more config values in the threatbus config.

- Only load configured plugins
- Provide more default values
- Update documentation with default values

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/threatbus](https://docs.tenzir.com/threatbus), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### CODE REVIEW

**Review by dominiklohmann — CHANGES_REQUESTED** [review]
This appears to work alright for Threat Bus itself, but what about the Threat Bus apps? E.g., there's a separate configuration file for PyVAST Threat Bus at `apps/vast/config.yaml.example`. These should be adapted as well.

File: README.md [hunk 1]
```diff
@@ -147,7 +152,7 @@ The integration tests require a local [Zeek][zeek] and
 [Docker](https://www.docker.com/) installation.
 
 
-## Plugin Development
+## Development
```
[the remark this answers is not in the document]
Comment (lava) [reply README.md:155]: I did not see what was specific about it; I follow the same steps for non-plugin threatbus development.

**Comment by lava** [comment]
@dominiklohmann Regarding the apps, these are technically separate from `threatbus` with their own configs. I've added commits to add default values for all settings in `pyvast-threatbus` and `suricata-threatbus` since these are actively used downstream, but `stix-shifter-threatbus` is more experimental/internal afaik so I'll leave that for a future PR.

File: README.md [hunk 2]
```diff
@@ -62,15 +62,20 @@ The following example shows how to connect [Zeek][zeek] via Threat Bus. There
 are more integrations available, so make sure to check out all
 [Threat Bus projects on PyPI](https://pypi.org/search/?q=threatbus).
 
+The example assumes that `threatbus` is available in your PATH. See the
+section on [Installation](#installation) below for more information on how to get there.
```
Comment (dominiklohmann) [inline README.md:66]: This needs text reflow to fit col 80.

**Review by dominiklohmann — APPROVED** [review]
I only tested Threat Bus and PyVAST Threat Bus locally, and they seem to work nicely. The changes to Suricata Threat Bus are similar enough and I trust that you tested them locally.

There's one text reflow issue in the README.

**Pull request merged**